spec/controllers/application_controller_spec.rb

```diff
@@ -101,40 +101,6 @@
     end
   end
 
-  context "#set_config" do
-    it "sets Processors details successfully" do
-      host_hardware = FactoryGirl.create(:hardware, :cpu_sockets => 2, :cpu_cores_per_socket => 4, :cpu_total_cores => 8)
-      host = FactoryGirl.create(:host, :hardware => host_hardware)
-      stub_user(:features => :all)
-
-      controller.send(:set_config, host)
-      expect(response.status).to eq(200)
-      expect(assigns(:devices)).to_not be_empty
-    end
-
-    it "doesn't crash on nil filename" do
-      disk = FactoryGirl.create(:disk, :filename => nil, :controller_type => nil, :device_type => 'disk', :mode => "foo")
-      host_hardware = FactoryGirl.create(:hardware, :cpu_sockets => 2, :cpu_cores_per_socket => 4, :cpu_total_cores => 8, :disks => [disk])
```

Comment:
This looks like we actually run some data through the `set_config`. I don't think we should just remove this. We should change this to call the new method and be more like a unit test.